node/subsystem/src/messages.rs

```diff
@@ -237,23 +237,44 @@ pub enum NetworkBridgeMessage {
 	/// Second parameter, tells what to do if we are not yet connected to the peer.
 	SendRequests(Vec<Requests>, IfDisconnected),
 
-	/// Connect to peers who represent the given `validator_ids`.
+	/// Connect to peers.
 	///
-	/// Also ask the network to stay connected to these peers at least
-	/// until the request is revoked.
-	/// This can be done by dropping the receiver.
-	ConnectToValidators {
-		/// Ids of the validators to connect to.
-		validator_ids: Vec<AuthorityDiscoveryId>,
-		/// The underlying protocol to use for this request.
+	/// Also ask the network to stay connected to these peers at least until the request is
+	/// revoked.  This can be done by dropping the corresponding `requests` sender.
+	ConnectPeers {
+		/// The underlying protocol to use for requests.
 		peer_set: PeerSet,
-		/// Response sender by which the issuer can learn the `PeerId`s of
-		/// the validators as they are connected.
-		/// The response is sent immediately for already connected peers.
-		connected: mpsc::Sender<(AuthorityDiscoveryId, PeerId)>,
+
+		/// Actual connection requests are gathered via this receiver.
+		///
+		/// It is possible to add more and more authorities we want to be connected to over time,
+		/// all managed by sending `ConnectToValidators` message. This is mostly useful if some
```

Comment:
```suggestion
		/// all managed by sending `ConnectMessage` message. This is mostly useful if some
```

Reply:
Oh that was not clear at all, I improved the docs a bit and will try to improve them further later.